test/ASTGen/types.swift

```diff
@@ -1,7 +1,8 @@
-// RUN: %target-typecheck-verify-swift -enable-experimental-feature ASTGenTypes
+// RUN: %target-typecheck-verify-swift -enable-experimental-feature ParserASTGen
 
 // -enable-experimental-feature requires an asserts build
 // REQUIRES: asserts
+// REQUIRES: rdar116686158
```

Comment:
It is really unfortunate that this test hits ASAN failure too. We're actively looking into this issue.
Anyway I confirmed these tests passes locally.

Reply:
Should we mark it `UNSUPPORTED: asan` for now so we at least don't lose the test coverage?